addons/website/static/src/scss/website.scss

```diff
@@ -2446,3 +2446,43 @@ input[value*="data-oe-translation-initial-sha"] {
 .o_dropdown_without_offset {
     transform: none !important;
 }
+
+// Subtle input-group-text
+// This can be set to an input-group-text placed directly before or after the input.
+.input-group-text.input-group-text-subtle {
+    @include transition($input-transition);
+    border-color: $input-border-color;
+    background: $input-bg;
+    color: $input-color;
```

Comment:
Same for the color property, it's already defined on the `.input-group-text`

Reply:
`.input-group-text` uses `$input-group-addon-color`. If the user changes this variable it will break the design. Here we need to have the same color of the input, `$input-color`.

Reply:
Good point, my bad sorry (ignore the above comment as well 👼 )